.changeset/popular-rice-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': patch
+---
+
+Optimized the use of the ProposalCore structure in the state getter
```

Comment:
I'm not sure we want a changelog for that, considering this is already new 5.0 code.

If we want a changelog entry, it would have to be more in line with the format we ususally use

Reply:
Yeah, I'd agree not to have a changelog entry in this case.